executor.c

```diff
@@ -10,11 +10,13 @@
 
 int executor(char **args, char **env)
 {
-	int success = 0;
-	
-	success = execve(args[0], args, env);
-	if (success < 0)
-		return (0);
+	pid_t childp = fork();
+
+	if (childp != 0)
+		wait(&childp);
+	else
+		if (execve(args[0], args, env) < 0)
+			return(0);
```

Comment:
try `exit` here, if u return the child will not terminate, it'll continue to the function that called executor